widgetastic_manageiq/__init__.py

```diff
@@ -603,6 +603,9 @@ class ParametrizedSummaryTable(ParametrizedView):
     PARAMETERS = ("title", )
     _table = SummaryTable(title=Parameter("title"))
 
+    def __locator__(self):
```

Comment:
PRT failures look directly related, as they're failing to find the Relationships table on provider details for stat validation.

Reply:
Tested the changes locally and observed the same failures. I tried a different fix by implementing is_displayed within ParametrizedSummaryTable and 

```
@property
    def is_displayed(self):
        return self._table.is_displayed
```
Failures were not observed with this fix. Maybe an alternate fix?